ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/InteractiveStatementExecutorTest.java

```diff
@@ -527,42 +496,42 @@ public void shouldSkipStartWhenReplayingLog() {
     final QueryId queryId = new QueryId("csas-query-id");
     final String name = "foo";
     final PersistentQueryMetadata mockQuery = mockReplayCSAS("CSAS", name, queryId);
-    mockQueryIdGenerator.activateNewGenerator(anyLong());
-    expectLastCall();
-    replayAll();
 
     // When:
     statementExecutorWithMocks.handleRestore(
         new QueuedCommand(
             new CommandId(Type.STREAM, name, Action.CREATE),
             new Command("CSAS", true, emptyMap(), emptyMap()),
             Optional.empty(),
-                0L
+            0L
         )
     );
 
     // Then:
-    verify(mockParser, mockEngine, mockQuery);
+    verify(mockQueryIdGenerator, times(1)).activateNewGenerator(anyLong());
   }
 
   @Test
+  @SuppressWarnings("unchecked")
```

Comment:
Out of curiosity, why was this added?

Reply:
Datasource takes a generic type and mockito yells at me when mocking generics. I haven't found a really good way around this so far.